pkg/vm/json_test.go

```diff
@@ -90,6 +92,30 @@ const (
 	testsDir = "testdata/neo-vm/tests/neo-vm.Tests/Tests/"
 )
 
+// stringToOpcode maps opcode's string representation to the opcode itself.
+var stringToOpcode = make(map[string]opcode.Opcode)
+
+func init() {
+	for i := 0; i < 255; i++ {
+		op := opcode.Opcode(i)
+		if s := op.String(); !strings.HasPrefix(s, "Opcode") {
+			stringToOpcode[s] = op
+		}
+	}
+	// Add opcodes with underscore
+	stringToOpcode["NEWARRAY_T"] = opcode.NEWARRAYT
+	stringToOpcode["CALL_L"] = opcode.CALLL
+	stringToOpcode["JMPEQ_L"] = opcode.JMPEQL
+	stringToOpcode["JMPNE_L"] = opcode.JMPNEL
+	stringToOpcode["JMPGE_L"] = opcode.JMPGEL
+	stringToOpcode["JMPGT_L"] = opcode.JMPGTL
+	stringToOpcode["JMPLE_L"] = opcode.JMPLEL
+	stringToOpcode["JMPLT_L"] = opcode.JMPLTL
+	stringToOpcode["JMPIFNOT_L"] = opcode.JMPIFNOTL
+	stringToOpcode["JMPIF_L"] = opcode.JMPIFL
+	stringToOpcode["JMP_L"] = opcode.JMPL
```

Comment:
It's better to add proper `FromString(string) Opcode` to `opcode` package. It might be useful in other places. And we can also change `String()` to use underscored versions.

Reply:
`String` is autogenerated, so we need to rename all constants to use underscore. This was done and declined in #907 or #901, because it makes linter unhappy.
And `FromString` is not that useful without corresponding `String` change. I used it only here, so decided not to add it to the opcode, but this is easy.

Reply:
Changing constants is not an option, but changing their string representations is. Not autogenerated, but given that nothing autogenerates `FromString`...

Reply:
Fixed.